.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+
+Thank you for your Pull Request!  We have developed this task checklist from the [Development Process Guide](https://roche-gsk.github.io/admiral/articles/development_process.html) to help with the final steps of the process.  Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   
+
+Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request.  This checklist is part of the Github Action workflows and the Pull Request will not be merged into the devel branch until you have checked off each task.
+
+- [ ] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
+- [ ] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://roche-gsk.github.io/admiral/articles/unit_test_guidance.html)
+- [ ] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://roche-gsk.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
+- [ ] Update to all relevant roxygen headers and examples 
+- [ ] Run `devtools::documents() ` so all `.Rd, NAMESPACE and DESCRIPTION` files are updated appropriately
```

Comment:
... so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are ...

`DESCRIPTION` shouldn't be touched by `devtools::document()`.

Reply:
@thomas-neitmann does this need updating in https://github.com/Roche-GSK/admiral/blob/devel/vignettes/development_process.Rmd also then? i included DESCRIPTION there - my mistake

Reply:
@rossfarrugia Yes I think the dev_process.Rmd should be updated.

Reply:
@thomas-neitmann , @rossfarrugia , @bms63 , `devtools::document()` updates the `Collate:` section in `DESCRIPTION`. Thus I would not remove it.

Reply:
@thomas-neitmann let me know what you think and i can put this back in dev process vignette as needed